docs/ldml/tr35-keyboards.md

```diff
@@ -977,6 +988,15 @@ _Attribute:_ `type` (required)
 
 > Current values: `simple`, `final`.
 
+
+There are other keying behaviors that are needed particularly in handling languages and scripts from various parts of the world. The behaviors intended to be covered by the transforms are:
```

Comment:
"handling languages and scripts from various parts of the worlds" seems a bit vague. How about shortening to "handing complex orthographies".

Reply:
updated